doc/generated/variables.gen

```diff
@@ -3097,7 +3097,6 @@ is <quote><literal>-dNOPAUSE -dBATCH -sDEVICE=pdfwrite</literal></quote>
         that Platform's logic will handle setting this value.
         This value is immutable, and should not be changed by the user after
         the Environment is initialized.
-        Currently only set for Win32.
```

Comment:
You need to make these two changes in `SCons/Platform/Platform.xml`. The `*.gen` files are generated so changing it here won't "stick".

Reply:
I changed it in both places. Should I undo the removals in this file?

Reply:
sorry, I managed to miss that when reading through :(   it's fine to leave the generated-file changes, I think, we sometimes update those in between releases but it's always regenerated and checked in at releast time, so either way works.